docs/api.yml

```diff
@@ -18,26 +18,59 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/metadata"
+                $ref: '#/components/schemas/metadata'
               example:
                 {
-                  "documentId": "kjsak2",
-                  "firstName": "The name",
-                  "dob": "1999-01-01",
+                  'documentId': 'kjsak2',
+                  'firstName': 'The name',
+                  'dob': '1999-01-01',
                 }
   /{documentId}:
     get:
       operationId: get-metadata
       summary: Gets a document from S3 using the document id
       parameters:
-        - $ref: "#/components/parameters/documentId"
+        - $ref: '#/components/parameters/documentId'
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/metadata"
+                $ref: '#/components/schemas/metadata'
+  /documents:
```

Comment:
I'm not sure about this URL, if I saw a POST to /documents - personally I'd expect that it'd be creating a document not searching them. Maybe we could do something like "/documents/search" or "/search"?